services/121-service/src/shared/middleware/azure-logger.middleware.ts

```diff
@@ -29,10 +29,21 @@ export class AzureLoggerMiddleware implements NestMiddleware {
         this.defaultClient.trackTrace({
           message: `${requestLog} - ${responseLog}}`,
         });
-        this.defaultClient.flush();
+        this.flushLogs();
       });
     }
 
     next();
   }
+
+  private flushLogs(): void {
+    this.defaultClient
+      .flush()
+      .then(() => {
+        return;
+      })
+      .catch(() => {
+        return;
```

Comment:
Should this .catch also do a console.error? (Like the other flushLogs method)

Reply:
in this case it wasn't part of a try/catch with a console.error in the catch. I tried to preserve current logic as much as possible. But we could add it anyway?

Reply:
If we can't think of a reason for why it wasn't there (I can't atm), then I'd say let's add it.